index.js

```diff
@@ -0,0 +1,12 @@
+var displayValue = function(val){
+  var value = val
+  return value
+
+}
+export {displayValue}
```

Comment:
there is not `export` in ES5. Look into binding to the `DOM Ready` event and then binding `click` and `keydown` handlers to DOM nodes of your calculator.